README.md

```diff
@@ -84,88 +84,84 @@ dependencies {
 
 #### iOS
 
-- 系统最低版本ios 8.0
+- Required minimum version iOS 8.0
 
 ```ruby
 source 'https://github.com/CocoaPods/Specs.git'
 platform :ios
 
 target 'MNNKitDemo' do
     platform :ios, '8.0'
-    
-    # 人脸检测
+
     pod 'MNNFaceDetection', '~> 0.0.1'
-    # 手势识别
     pod 'MNNHandGestureDetection', '~> 0.0.1'
-    # 人像分割
     pod 'MNNPortraitSegmentation', '~> 0.0.1'
-    
+
 end
 ```
 
 ##### Bitcode
 
-目前MNNKit SDK均不支持bitcode，应用集成时候需关闭bitcode选项。
-
+MNNKit SDKs doesn't support Bitcode and you'll need to turn off Bitcode in your application.
 
 
 ## API
 
-Kit的基本API只有三个，创建实例、推理、释放实例。使用流程也是按照这个顺序，如下图所示，其中推理时可输入视频、图片或其他格式的数据。
+There are only three basic APIs in the kit:
+  - Create Instance
+  - Inference
+  - Release Instance
+And this is also the sequence for using it, as shown in the graphs below. You can use video/image or other formats of data as the input of the inferencing stage.
 
 ![api](doc/api_flow.png)
 
-[人脸检测API](doc/FaceDetection_CN.md)
-
-[手势识别API](doc/HandGestureDetection_CN.md)
+[FaceDetection API](doc/FaceDetection_CN.md)
 
-[人像分割API](doc/PortraitSegmentation_CN.md)
+[HandGestureDetection API](doc/HandGestureDetection_CN.md)
 
+[PortraitSegmentation API](doc/PortraitSegmentation_CN.md)
 
 
-## 接入指南
 
-MNNKit API中包含了inAngle和outAngle两个参数值，他们是用来做什么的？理解这个问题之前，我们不妨先了解下SDK处理的一般过程。
+## Integration Guide
 
-### 处理过程
+MNNKit API contains two arguments: ``inAngle`` and ``outAngle``. What are they used for? Because understanding this question, we should first understanding the generic process of how the SDK works.
+### Overview
 
-如下是iOS和Android设备后置摄像头正向拍摄的场景，端上整个处理过程：
+Below is the whole process of processing what the back camera of iOS and Android devices has captured.
 
 ![process](doc/process_graph.png)
 
-#### 1. 设备预览和输出
+#### 1. Device preview and output
 
-这部分是从Camera获取数据，作为SDK的输入；同时在设备前端，要正确的预览给用户。
+This stage captures data from the camera and feed it into the SDK while providing proper preview on the screen.
 
-#### 2. SDK检测
+#### 2. SDK Detection
 
-MNNKit SDK内部实现封装了算法的深度学习模型，运行在MNN引擎上，因此人脸检测也并非传统的图像处理，它有着机器学习所有的特性。算法在模型训练阶段使用的是正向的人脸数据集，因而这个模型只认识"正向"的人脸，如果输入是一张方向不对的人脸，可能就不会有检测结果，我们称之为"内容方向敏感"。MNNKit中人脸检测、手势识别、人像分割都属于"内容方向敏感"的类型。
+MNNKit SDK contains the deep learning model of related algorithms and runs on top of MNN engine. As a result it's not traditional image processing and contains all the characteristics of deep learning. During training stage algorithm uses face images in top-down direction as such this model only recognize faces in that same direction. If the input contains faces in a different direction, it might not get detected. We call this ``Content Orientation Sensitive``. In MNNKit, FaceDetection,HandGestureDetection and PortraitSegmentation are all ``Content Orientation Sensitive``.
```

Comment:
```suggestion
MNNKit SDK contains the deep learning models of related algorithms and runs on top of MNN engine. As a result it's not traditional image processing and contains all the characteristics of deep learning. During the training stage, the algorithm uses face images in upright orientation. As a result, this model only recognizes faces in that orientation. If the input contains faces in a different orientation, it might not get detected. We call this ``Content Orientation Sensitive``. In MNNKit, FaceDetection,HandGestureDetection and PortraitSegmentation are all ``Content Orientation Sensitive``.
```